src/Components/ClcScrn.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ClcScrn extends React.PureComponent {
+  render() {
+    const { value } = this.props;
+    return (
+      <div className="clcScrn">
+        {value}
+      </div>
+    );
+  }
+}
+
+ClcScrn.propTypes = {
+  value: PropTypes.string.isRequired,
+};
+
+export default ClcScrn;
```

Comment:
### Although I am highlighting this file, the changes I am about to suggest should be made to files that had no code changes:

- [ ] One of the requirements is to delete all files that were created with the create react command that won't be relevant to this project. In light of this, kindly delete the following files:

- In the `public` directory:
  - favicon.ico
  - logo192.png
  - logo512.png
  - manifest.json
  - robots.txt
- In the `src` directory:
  - App.test.js
  - logo.svg
  - reportWebVitals.js
  - setupTests.js